Frontend/src/pages/admin/CreateNotificationPage.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <q-card flat bordered id="card">
+    <q-card-section>
+      <div class="text-h6">Send Notification</div>
+    </q-card-section>
+
+    <q-separator inset />
```

Comment:
Wrap this in BasePage as well

Reply:
uhhhhhhhhhh this file was supposed to be deleted im confused